Remotion/ObjectBinding/UnitTests/AvailableResourcesTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using NUnit.Framework;
+using Remotion.Development.Globalization;
+using Remotion.Globalization;
+
+namespace Remotion.ObjectBinding.UnitTests
+{
+  [TestFixture]
+  public class AvailableResourcesTest
+  {
+    [Test]
+    public void AllAvailableResourcesAreDeclaredCorrectlyAndHaveTheirResources ()
+    {
+      AvailableResourcesTester.CheckAvailableResources<AvailableResourcesLanguagesAttribute>(typeof(BusinessObject).Assembly);
```

Comment:
why not hardcode the attribute?

Reply:
Because I did not want to introduce a globalization reference in the shared source project and passing the attribute seems like reasonable compromise that won't break instantly on rename/move of the attribute

Reply:
Ah, I see. hmm... got an idea; we could link the Attribute over into the shared files project. That way, we can simplify the code :)